drivers/exec/driver_test.go

```diff
@@ -273,7 +275,7 @@ func TestExecDriver_StartWaitRecover(t *testing.T) {
 // task dies, the orphans in the PID namespaces are killed by the kernel
 func TestExecDriver_NoOrphans(t *testing.T) {
 	t.Parallel()
-	require := require.New(t)
+	r := require.New(t)
```

Comment:
are we doing this now?

Reply:
I only care that we do not shadow names. Whether we use `require.Blah(t, ...` or rename `require` as a variable or something else, I don't have much of an opinion.